Flow.Launcher/ResultListBox.xaml

```diff
@@ -29,41 +30,43 @@
                     </ControlTemplate>
                 </Button.Template>
                 <Button.Content>
-                    <Grid HorizontalAlignment="Stretch" VerticalAlignment="Stretch" Margin="5"
+                    <Grid HorizontalAlignment="Stretch" VerticalAlignment="Stretch" Margin="0"
                           Cursor="Hand" UseLayoutRounding="False">
                         <Grid.Resources>
                             <converter:HighlightTextConverter x:Key="HighlightTextConverter"/>
                             <converter:OrdinalConverter x:Key="OrdinalConverter" />
                             <converter:OpenResultHotkeyVisibilityConverter x:Key="OpenResultHotkeyVisibilityConverter" />
                         </Grid.Resources>
                         <Grid.ColumnDefinitions>
-                            <ColumnDefinition Width="32" />
-                            <ColumnDefinition />
-                            <ColumnDefinition Width="0" />
+                            <ColumnDefinition Width="54" />
+                            <ColumnDefinition Width="*"/>
```

Comment:
![image](https://user-images.githubusercontent.com/6903107/135047708-46fff705-4b60-4be8-8aee-ff6cc9fca0a2.png)
I changed list item layout. It looks good and solves the problem that the Title area invades the hot key area.